### PR TITLE
Misc fromRValues cleanup

### PR DESCRIPTION
**Pull request by alissapajer**
[ch437]

### CODE REVIEW

**Comment by alissapajer** [comment]
I created ch1051 to track removing `fromRValues` from production and replacing it with a test function that delegates to `fromRValueStream`.

File: yggdrasil/src/main/scala/quasar/yggdrasil/table/Slice.scala [hunk 1]
```diff
@@ -2145,7 +2145,9 @@ object Slice {
     val maxColumnsC = maxColumns.getOrElse(Config.maxSliceColumns)
     // println(s"maxRows: $maxRowsC, maxCols: $maxColumnsC")
 
-    def rec(next: ArraySliced[RValue], values: fs2.Stream[F, RValue]): fs2.Pull[F, Slice, Unit] =
+    // FIXME not tailrec
```
Comment (djspiewak) [inline yggdrasil/src/main/scala/quasar/yggdrasil/table/Slice.scala]: It is actually. It's tailrec through the suspension – ironically, this was the bug that I fixed back from when it was using for-comprehensions! Regular tailrec wouldn't actually make any sense here.

File: yggdrasil/src/test/scala/quasar/yggdrasil/TestIdentities.scala [hunk 2]
```diff
@@ -23,9 +23,11 @@ import quasar.precog.common._
 import cats.effect.IO
 import scalaz._, Scalaz._, Ordering._
 
-package object yggdrasil {
-  type Identity   = Long
+object TestIdentities {
```
Comment (djspiewak) [inline yggdrasil/src/test/scala/quasar/yggdrasil/TestIdentities.scala]: Outstanding.

File: yggdrasil/src/main/scala/quasar/yggdrasil/package.scala [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+package quasar 
+
+import quasar.contrib.scalaz.MonadTell_
+
+package object yggdrasil {
+
+  type Finalizers[F[_]] = List[F[Unit]]
+
+  type MonadFinalizers[F[_], G[_]] = MonadTell_[F, Finalizers[G]]
```
Comment (wemrysi) [inline yggdrasil/src/main/scala/quasar/yggdrasil/package.scala]: :heart:

**Comment by alissapajer** [comment]
@djspiewak ready! :)

**Pull request merged**